api_plugin/src/server.cpp

```diff
@@ -1,31 +1,29 @@
 #include "irods/plugins/api/private/genquery2_common.hpp"
 #include "irods/plugins/api/genquery2_common.h" // For API plugin number.
 
+#include "irods/genquery2_driver.hpp"
+#include "irods/genquery2_sql.hpp"
+
 #include <irods/apiHandler.hpp>
-#include <irods/catalog.hpp>           // Requires linking against libnanodbc.so
-#include <irods/catalog_utilities.hpp> // Requires linking against libnanodbc.so
-#include <irods/irods_logger.hpp>
+#include <irods/irods_configuration_keywords.hpp>
+#include <irods/irods_get_full_path_for_config_file.hpp>
 #include <irods/irods_rs_comm_query.hpp>
-//#include <irods/irods_re_serialization.hpp> // For custom data types that can be used in the NREP.
+#include <irods/irods_server_properties.hpp>
 #include <irods/irods_server_properties.hpp>
 #include <irods/procApiRequest.h>
 #include <irods/rodsConnect.h>
 #include <irods/rodsDef.h>
 #include <irods/rodsErrorTable.h>
```

Comment:
Aren't these 4.3.x paths?

Reply:
No. They look that way because the project doesn't include `/usr/include/irods` as an include path.

Reply:
Understood. Thanks!

Reply:
This is how external projects should reference iRODS headers. You can get around this by doing something like ...
```cmake
target_include_directories(${THING} PRIVATE ... ${IRODS_INCLUDE_DIRS} ...)
```